common/tests/refcount_ut/refcount_ut.c

```diff
@@ -244,7 +244,7 @@ BEGIN_TEST_SUITE(TEST_SUITE_NAME_FROM_CMAKE)
         ASSERT_ARE_EQUAL(char_ptr, umock_c_get_expected_calls(), umock_c_get_actual_calls());
     }
 
-    TEST_FUNCTION(refcount_INC_REF_and_DEC_REF_after_create_says_we_should_not_free)
+    TEST_FUNCTION(refcount_INC_REF_and_DEC_REF_after_create_says_we_should_not_free) // no-srs

```

Comment:
> // no-srs [](http://example.com/codeflow?start=84&length=10)

There should be Tests tags with specs that are tested for this function. And the ones below in the file.

Reply:
Done. Added Tests_SRS_REFCOUNT_01_008 tags to the tests.